lucien-ios/AddNewComicBook/AddComicViewController.swift

```diff
@@ -10,10 +10,11 @@ import UIKit
 import QuartzCore
 import AVFoundation
 
-final class AddComicViewController: UIViewController, UIPickerViewDelegate, UIPickerViewDataSource, UITextFieldDelegate {
+final class AddComicViewController: UIViewController, UIPickerViewDelegate, UIPickerViewDataSource, UITextFieldDelegate, CameraViewControllerDelegate, UIScrollViewDelegate {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Type Body Length Violation: Type body should span 200 lines or less excluding comments and whitespace: currently spans 218 lines [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Atype_body_length)